lessons/git/git-collaboration-2in1/index.md

```diff
@@ -0,0 +1,291 @@
+Co je to vlastně ten zmíněný *repozitář*? Můžeme ho vytvořit z jakéhokoliv adresáře na disku, kde máme uložené soubory, 
+jen Gitu řekneme, že z něj chceme ten repozitář udělat. To má za následek:
+
+1. Git bude sledovat změny, které ve složce uděláme.
+2. Budeme schopni vytvořit verzi tohoto adresáře, ke které se můžeme kdykoliv vrátit.
+3. Můžeme nastavit vzdálené repozitáře, kam chceme změny nahrávat nebo naopak stahovat změny od jiných lidí. 
+
+!!! note "Pozor"
+    Budeme hodně pracovat s příkazovou řádkou. Jestli se s ní ještě nekamarádíš, koukni se na [úvod](../../beginners/cmdline/).
+    Nezapomeň: $ na začátku se nepíše; je tu proto, aby šlo poznat že jde o příkaz.
+
+Naším dnešním cílem je ale přispět do projektu, který založil někdo jiný. Proto použijeme příkaz `git clone` ke stažení vzdáleného repozitáře.
+
+    $ git clone https://github.com/asgeirrr/prezencka
+
+V aktuálním adresáři se nám vytvořil podadresář `prezencka`, který obsahuje všechny soubory, které vidíme ve webovém rozhraní na [GitHubu](https://github.com/asgeirrr/prezencka).
+
+
+#### Stav repozitáře (`git status`)
+
+Velmi často potřebujeme zjistit současný stav repozitáře. Navigujte v příkazové řádce do složky `prezencka` a zadejte `git status`. Git nám odpoví:
+
+    $ git status
+    On branch master
+    Your branch is up-to-date with 'origin/master'.
+    nothing to commit, working tree clean
+
+Git nám v tuto chvíli poskytl 3 informace:
+
+1. Že jsme na větvi `master`. Git umožňuje pracovat na více věcích současně pomocí tzn. *větví* mezi kterými můžeme přepínat. Pak se na na disku "objeví" verze souborů v dané větvi. My si dnes vystačíme s jednou hlavní větví, která se tradičně jmenuje `master`.
+2. Že naše větev `master` je aktuální vůči větvi na vzdáleném repozitáři (origin) na GitHubu.
+3. Že jsme neudělali žádné změny oproti poslední verzi souborů, což je pravda.
+
+#### První revize (`git commit`)
+
+Už jsme několikrát zmínili, že Git je verzovací systém. Jak že to ale soubory verzuje? 
+V podstatě mu musíme dát příkaz, aby zmrazil současný stav repozitáře, tzv. vytvořit *revizi*.
+Pak můžeme dělat další změny, ale už vždy budeme schopni se k této revizi vrátit.
+
+Přidejte do repozitáře soubor `vase_jmeno.txt`, např. tedy `magdalena_rettigova.txt`
+a do něj napište svoje současné povolání nebo cokoliv jiného.
+Zkontrolujte současný stav repozitář pomocí `git status`: Git oznámí, že v adresáři je soubor, o kterém ještě „neví“. 
+
+    $ git status
+    On branch master
+    Your branch is up-to-date with 'origin/master'.
+    Untracked files:
+      (use "git add <file>..." to include in what will be committed)
+
+          magdalena_rettigova.txt
+
+    nothing added to commit but untracked files present (use "git add" to track)
+
+U každého nového souboru musíme Gitu říct, že chceme jeho obsah sledovat. Proveďte to se svým souborem:
+
+    $ git add magdalena_rettigova.txt
```

Comment:
Myslím, že motivace @encukou pro tohle byla minimalizovat počet přepínačů, které se musí učit a taky to, aby to bylo víc "ruční".